### DISCUSSION TITLE
You tube video download failed. States free with ads, not sure if that is reason.

### DISCUSSION THREAD

**Discussion by Logan82461** [aclap-dev/video-downloadhelper#592] [Help and Q&A]
### Your question

I believe I have downloaded some other titles that said free with ads so  not sure if that is what is giving me an issue here or not.
I appreciate any help. 
Just started using the software again after a few years so everything is a little knew. Sorta miss when it was attached to your tool bar at top, but it seems to work fine as well on the sidebar.
Here is the error code generated the the link to the video. Really long error code. 

https://www.youtube.com/watch?v=5O4eaxJJ-Pc

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.youtube.com/watch?v=5O4eaxJJ-Pc",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 000002844f74b080] Unable to open key file skd://www.youtube.com/api/drm/fps?ek=c29689026fd25d0a8bb6c2040d6ed082\r\n[h264 @ 000002844fe8d4c0] top block unavailable for requested intra mode\r\n[h264 @ 000002844fe8d4c0] error while decoding MB 3 0, bytestream 51627\r\n[hls @ 000002844fe787c0] Unable to open key file skd://www.youtube.com/api/drm/fps?ek=d4ca00c860a256bfb62fa9ffc5cc20b6\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (15) exceeds limit (14).\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (60) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (16) exceeds limit (14).\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (19) exceeds limit (14).\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (73) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (58) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (63) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (62) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (75) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (69) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (62) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (62) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (60) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (70) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (63) exceeds limit (49).\r\n    Last message repeated 1 times\r\n[aac @ 000002845056a380] TNS filter order 22 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 000002845056a380] channel element 1.12 is not allocated\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (64) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (50) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (54) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (69) exceeds limit (49).\r\n[aac @ 000002845056a380] channel element 2.7 is not allocated\r\n[aac @ 000002845056a380] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (63) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 16 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (54) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (56) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (58) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (60) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 000002845056a380] invalid band type\r\n    Last message repeated 1 times\r\n[aac @ 000002845056a380] Number of bands (78) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 24 is greater than maximum 12.\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (64) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (70) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (55) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (52) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (67) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (67) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (54) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 23 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (75) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (63) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (67) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (56) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (54) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (62) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (62) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (58) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (64) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (66) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (67) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 15 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (69) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (64) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (56) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (59) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (63) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (68) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 19 is greater than maximum 12.\r\n[aac @ 000002845056a380] error in spectral data, ESC overflow\r\n[aac @ 000002845056a380] TNS filter order 17 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (56) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 25 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (68) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (72) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (54) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (60) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 27 is greater than maximum 12.\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (66) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] TNS filter order 21 is greater than maximum 12.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 31 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 14 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (69) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (65) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (70) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (53) exceeds limit (49).\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Number of bands (50) exceeds limit (49).\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] TNS filter order 30 is greater than maximum 12.\r\n[aac @ 000002845056a380] invalid band type\r\n[aac @ 000002845056a380] Pulse data corrupt or invalid.\r\n[aac @ 000002845056a380] Number of bands (57) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (64) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 13 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 28 is greater than maximum 12.\r\n[aac @ 000002845056a380] TNS filter order 20 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (61) exceeds limit (49).\r\n[aac @ 000002845056a380] TNS filter order 18 is greater than maximum 12.\r\n[aac @ 000002845056a380] Number of bands (51) exceeds limit (49).\r\n[aac @ 000002845056a380] Number of bands (56) exceeds limit (49).\r\n[mp4 @ 0000028450ac40c0] sample rate not set\r\n[out#0/mp4 @ 000002844fea3780] Could not write header (incorrect codec parameters ?): Invalid argument\r\nError opening output file D:\\Movies\\Dracula Dead and Loving It.mp4.\r\nError opening output files: Invalid argument\r\n",
    "id": "coapp_failure_420724ef-0396-4fc2-89be-0a5f32e10d2b",
    "downloadable_id": "downloadable_555809307237609"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"92e2053001f94f0f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "D:\\Movies",
    "theme": "system",
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": true,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 2
}

**Comment by paulrouget** [comment]
What you're downloading here is DRM content. Some videos are like that. Sadly, we do not support DRM: https://github.com/aclap-dev/video-downloadhelper/wiki/What-can-I-do-if-the-video-is-protected-by-DRM%3F

**Reply by GrampaWildWilly** [reply]
Yes.  Firefox tells you about this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/336339c4-5066-406d-ba24-3df28b7f5dd9)

Just hover your mouse over the indicated icon.  When there is no DRM protection on an item, that icon is not present.

This works ONLY on Firefox.  It is my understanding that neither Chrome nor Edge offers this hint.

**Reply by Logan82461** [reply]
thanks, :)
